targets/targets.json

```diff
@@ -3791,7 +3795,7 @@
             "STM32F746NG",
             "STM_EMAC"
         ],
-        "components_add": ["QSPIF"],
+        "components_add": ["QSPIF", "SD", "FLASHIAP"],
```

Comment:
SD seems not correct ?

Reply:
Hi @jeromecoutant,
How do you know it is not correct?

Reply:
Because board doesn't have any embedded SD connector over SPI